tests/integration-tests/jobs/createJobKubernetes-test.js

```diff
@@ -136,6 +138,92 @@ describe('Create job specific kubernetes tests', async function () {
+                                metadata: { name: 'jobName', uid: 'uid' },
+                                namespace: kubernetesConfig.kubernetesNamespace
+                            });
+
+                        const jobBody = {
+                            test_id: testId,
+                            arrival_rate: 1,
+                            duration: 1,
+                            environment: 'test',
+                            run_immediately: true,
+                            type: 'load_test',
+                            experiments: [
+                                {
+                                    experiment_id: chaosExperimentId,
+                                    start_after: 5000
+                                }
+                            ]
+                        };
+
+                        createJobResponse = await schedulerRequestCreator.createJob(jobBody, {
+                            'Content-Type': 'application/json'
+                        });
+
+                        should(createJobResponse.status).eql(201);
+                        oneTimeJobId = createJobResponse.body.id;
+                    });
+
+                    it('Create second job which is cron functional_test', async () => {
+                        const jobBody = {
+                            test_id: testId,
+                            arrival_count: 1,
+                            duration: 1,
+                            environment: 'test',
+                            run_immediately: false,
+                            cron_expression: '* 10 * * * *',
+                            type: 'functional_test',
+                            experiments: [
+                                {
+                                    experiment_id: chaosExperimentId,
+                                    start_after: 5000
+                                }
+                            ]
+                        };
+
+                        createJobResponse = await schedulerRequestCreator.createJob(jobBody, {
+                            'Content-Type': 'application/json'
+                        });
+
+                        should(createJobResponse.status).eql(201);
+                        cronJobId = createJobResponse.body.id;
+                    });
+
+                    it('Get the jobs, without one_time query param, only cron job should be returned', async () => {
+                        getJobsFromService = await schedulerRequestCreator.getJobs({
+                            'Content-Type': 'application/json'
+                        });
+
+                        should(getJobsFromService.status).eql(200);
+
+                        const relevantJobs = getJobsFromService = getJobsFromService.body.filter(job => job.id === cronJobId || job.id === oneTimeJobId);
+                        should(relevantJobs.length).eql(1);
+                        should(relevantJobs[0].id).eql(cronJobId);
```

Comment:
what exactly do you intend to check in this test? That the API still works well? Or do these tests should be extended when additional logics enters?